File: WowUp.Common/Models/PotentialAddon.cs
namespace WowUp.Common.Models
{
    public class PotentialAddon
    {
        public string Name { get; set; }
        public string ProviderName { get; set; }
        public string ThumbnailUrl { get; set; }
        public string ExternalId { get; set; }
        public string ExternalUrl { get; set; }
        public string Author { get; set; }
        public int DownloadCount { get; set; }
    }
}

File: wowup-electron/src/app/models/wowup/potential-addon.ts
export interface PotentialAddon {
  name: string;
  providerName: string;
  thumbnailUrl: string;
  screenshotUrl?: string;
  externalId: string;
  externalUrl: string;
  author: string;
  downloadCount: number;
  summary?: string;
  screenshotUrls?: string[];
  version?: string;
}
